src/Voron/Data/Tables/Table.cs

```diff
@@ -1680,6 +1680,25 @@ public long DeleteBackwardFrom(TableSchema.FixedSizeSchemaIndexDef index, long v
             return deleted;
         }
 
+        public bool FindByIndex(TableSchema.FixedSizeSchemaIndexDef index, long value, out TableValueReader reader)
+        {
+            AssertWritableTable();
+            reader = default;
```

Comment:
This will create `TableValueReader ` instance even if we return `true`. It's minor but can we avoid that and create it only in cases where `false` is returned